python/lsst/ts/cRIOpy/M1M3/HardpointsWidget.py

```diff
@@ -421,6 +422,20 @@ def hardpointActuatorSettings(self, data):
 
     @Slot(map)
     def hardpointActuatorData(self, data):
+        hs = self.m1m3.remote.evt_hardpointActuatorSettings.get()
+        if hs is not None:
+            for idx, f in enumerate(data.measuredForce):
+                color = Colors.OK
+                if f < hs.hardpointMeasuredForceWarningLow or f > hs.hardpointMeasuredForceWarningHigh:
+                    color = Colors.WARNING
+                if f < hs.hardpointMeasuredForceFaultLow or f > hs.hardpointMeasuredForceFaultHigh:
```

Comment:
Same comment above.

Reply:
Surprisingly it passed through old version of black - new reformat it as expected.